src/convertToObject.js

```diff
@@ -6,7 +6,16 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arrayWithStyles = complexStylesString.split(';')
```

Comment:
The parameter `sourceString` is defined but not used in the function. You should replace `complexStylesString` with `sourceString` to utilize the function parameter correctly.